data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/CountAggregateAction.java

```diff
@@ -56,13 +61,25 @@ private long getTimeNanos(Instant time) {
         return currentTimeNanos;
     }
 
+    public Exemplar createExemplar(final Event event) {
+        long curTimeNanos = getTimeNanos(Instant.now());
+        Map<String, Object> attributes = event.toMap();
```

Comment:
I'm not sure this is exactly what the attributes is meant to convey. In the model these are [`filtered_attributes`](https://github.com/open-telemetry/opentelemetry-proto/blob/81a296f9dba23e32d77f46d58c8ea4244a2157a6/opentelemetry/proto/metrics/v1/metrics.proto#L648-L651
).

And the current code in Data Prepper for metric exemplars uses only the provided `filtered_attributes`.

https://github.com/opensearch-project/data-prepper/blob/0c7b7cba734137cdc503bd7437b8b20c324e9376/data-prepper-plugins/otel-metrics-raw-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/otelmetrics/OTelMetricsProtoHelper.java#L326

Reply:
Yeah. I do not know why SAP folks called it Attributes in the DataPrepper implementation